src/components/VideoCard/VideoCard.component.jsx

```diff
@@ -0,0 +1,68 @@
+import React, { useState } from 'react';
+import Grid from '@material-ui/core/Grid';
+import Card from '@material-ui/core/Card';
+import ButtonBase from '@material-ui/core/ButtonBase';
+import { Link } from 'react-router-dom';
+import CardMedia from '@material-ui/core/CardMedia';
+import CardHeader from '@material-ui/core/CardHeader';
+import Avatar from '@material-ui/core/Avatar';
+import { channels } from '../../dummyData/videos';
```

Comment:
Why did you use this dummy data instead of the real channel data that your component receives on props?